src/applications/static-pages/homepage-hero-randomizer/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import './sass/homepage-hero.scss';
+
+// shuffle array using Fisher-Yates shuffle
+const randomizeOrder = imageArray => {
+  const tempArray = imageArray;
+  let currentIndex = Object.keys(tempArray).length;
+  let randomIndex;
+
+  // While there remain elements to shuffle...
+  while (currentIndex !== 0) {
+    // Pick a remaining element...
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex--;
+
+    // And swap it with the current element.
+    [tempArray[currentIndex], tempArray[randomIndex]] = [
+      tempArray[randomIndex],
+      tempArray[currentIndex],
+    ];
+  }
+
+  // grab first 3 items
+  return tempArray.slice(0, 3);
+};
+
+const HeroRandom = () => {
```

Comment:
I'm not sure an `id` property is necessary here. As such, I think this array can be altered to be an array of strings rather than objects.